app/src/main/java/com/duckduckgo/app/onboarding/ui/page/WelcomePage.kt

```diff
@@ -50,6 +54,13 @@ class WelcomePage : OnboardingPageFragment() {
         applyFullScreenFlags()
     }
 
+    override fun onPause() {
```

Comment:
should we be doing this `onDestroy` instead of `onPause`?

Reply:
Yeah, makes sense!